packages/dose3/dose3.6.0.1/opam

```diff
@@ -0,0 +1,71 @@
+opam-version: "2.0"
+synopsis: "Dose library (part of Mancoosi tools)"
+description: """
+The dose suite provides libraries for handling package meta-data, and various
+ tools for analyzing package relationships in a large package repository.
+  - dose-builddebcheck checks, given a collection of source package stanzas
+    and a collection of binary package stanzas of Debian packages, whether
+    the build-dependencies of each source package can be satisfied by the
+    binary packages.
+  - dose-distcheck checks for every package of a distribution whether it
+    is possible to satisfy its dependencies and conflicts within this
+    distribution.
+  - ceve, a general metadata parser supporting different input formats
+    (Debian, rpm, and others) and different output formats.
+  - dose-outdated, a Debian-specific tool for finding packages that are not
+    installable with respect to a package repository, and that can only be
+    made installable again by fixing the package itself.
+  - dose-challenged, a Debian-specific tool for checking which packages
+    will certainly become uninstallable when some existing package is upgraded
+    to a newer version.
+  - dose-deb-coinstall, a Debian-specific tool for checking whether a set of
+    packages can be installed all together."""
+maintainer: [
+  "Pietro Abate" "Johannes Schauer Marin Rodrigues" "Ralf Treinen"
+]
+authors: [
+  "Pietro Abate"
+  "Ralf Treinen"
+  "Jaap Boender"
+  "Johannes Schauer Marin Rodrigues"
+  "Roberto Di Cosmo"
+  "Felipe Garay"
+  "Stefano Zacchiroli"
+  "Jakub Zwolakowski"
+  "Olivier Rosello"
+]
+license: "LGPL-3.0-or-later with OCaml-LGPL-linking-exception"
+homepage: "http://www.mancoosi.org/software/"
+doc: "https://irill.gitlab.io/dose3"
+bug-reports: "https://gitlab.com/irill/dose3/issues/"
+dev-repo: "git+https://gitlab.com/irill/dose3.git"
+depends: [
+  "ocaml" {>= "4.07"}
+  "dune" {>= "2.7.1"}
```

Comment:
The top of the dune-file suggests that you only need dune 2.3, do you want to relax this bound?
```suggestion
  "dune" {>= "2.7.1"}
```